bgpd/bgp_nht.c

```diff
@@ -881,7 +881,7 @@ void bgp_nht_register_enhe_capability_interfaces(struct peer *peer)
 
 	if (!sockunion2hostprefix(&peer->su, &p)) {
 		if (BGP_DEBUG(nht, NHT))
-			zlog_debug("%s: Unable to convert prefix to sockunion",
+			zlog_debug("%s: Unable to convert sockunion to prefix",
```

Comment:
only one thing from me: shouldn't this be a warning instead of debug?

Reply:
Since the equivalent message on the register side had been a debug, I kept the same on the deregister side.  I'm not certain if this should be moved to warning and removed from requiring the debug to be enabled or not.  If you believe it needs to be a warning, I'll make the change.

Reply:
what do others think?

Reply:
warning/error may be more appropriate here.

Reply:
I asked Donald his thoughts on it and he said it was fine staying as a debug.  I'm also fine changing it if that's the consensus.

Reply:
I'm fine with either... I guess the question would be -- what would the user do differently if it were a warning rather than a debug? If the session doesn't come up the way I expected, I'd be looking at debugs anyway?